docs.json

```diff
@@ -127,6 +127,10 @@
       "group": "Advanced",
       "tab": "documentation",
       "pages": [
+        {
+          "title": "Physical iOS devices",
```

Comment:
I would change this title to smth like Setup for physical iOS devices. Without context of setup instruction it's not that obvious